ansible/roles/riakcs/defaults/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+riak_user: riak
+
+riak_package_release: 1
+riak_version: 2.1.1
+riak_script_url: https://packagecloud.io/install/repositories/basho/riak/script.deb.sh
+riak_install_script: /tmp/riak-install
+riak_dest: /etc/riak
+riak_data_dir: /opt/data/riak
+riak_conf: riak.conf
+
+riakcs_package_release: 1
+riakcs_version: 2.0.1
+riakcs_ebin: /usr/lib/riak-cs/lib/riak_cs-{{ riakcs_version }}/ebin
+riakcs_script_url: https://packagecloud.io/install/repositories/basho/riak-cs/script.deb.sh
+riakcs_install_script: /tmp/riakcs-install
+riakcs_dest: /etc/riak-cs
+riakcs_conf: riak-cs.conf
+# Where to write the riakcs key/secret for the admin user
+riakcs_keys_dest: /home/salt/ansible/config/{{ deploy_env }}/{{ deploy_env }}.yml
```

Comment:
wasn't sure how to best do this. my guess is that every config will override this variable. it's a useful function to have otherwise you have to log into the machine and open the conf to get the admin keys. (could possibly make it relative)